dataland-frontend/tests/component/components/resources/frameworkDataSearch/ThreeLayerDataTable.cy.ts

```diff
@@ -218,4 +227,63 @@ describe("Component test for the NewEUTaxonomy Page", () => {
       cy.get("table").find(`tr:contains(${capexFirstNonAlignedActivityAbsoluteShare})`);
     });
   });
+
+  it("Check P2p view page that properties with same name in different KPIs have different values", () => {
+    const metaInfo: DataMetaInformation = {
+      companyId: "some-fake-ID-123456789",
+      dataId: "some-fake-dataId-123456789",
+      dataType: DataTypeEnum.P2p,
+      reportingPeriod: "2018",
+      currentlyActive: true,
+      qaStatus: "Accepted",
+      uploadTime: 1234567890,
+    };
+
+    const p2pData: PathwaysToParisData = {
+      general: {
+        general: {
+          dataDate: "2023-09-30",
+          sectors: [P2pSector.Ammonia, P2pSector.Cement],
+        },
+        governance: {},
+        climateTargets: {},
+        emissionsPlanning: {},
+        investmentPlanning: {},
+      },
+      ammonia: {
+        decarbonisation: {
+          energyMix: 11,
+        },
+        defossilisation: {},
+      },
+      cement: {
+        energy: {
+          energyMix: 22,
+        },
+        technology: {},
+        material: {},
+      },
+    };
+
+    cy.mountWithPlugins(ThreeLayerDataTable, {
+      keycloak: minimalKeycloakMock({}),
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
Removing this disable and running the linter yields no errors locally. Why is it required?

Reply:
I get errors locally:

![image](https://github.com/d-fine/Dataland/assets/131248192/b4563010-b49e-4976-bcbb-359ae8b257c6)

![image](https://github.com/d-fine/Dataland/assets/131248192/cc9833ae-dcec-423f-8a0e-05e53713d8a8)

Reply:
Sorry I was not clear in my comment. I meant removing both comment lines. The eslint disable and the ts ignore. When I do that locally the linter is still fine and the component tests are still green.

Reply:
Removing both I get an even chunkier message :)
![image](https://github.com/d-fine/Dataland/assets/131248192/c645c833-b216-43ce-9165-68e0f5d4fbe6)

Marc mentioned this during his overview of component testing that these comments are necessary to shut typescript up.

Reply:
I am wondering why I do not get any errors locally. But for this PR it does not matter.